docs/core_concepts/16_roles_and_permissions/index.mdx

```diff
@@ -0,0 +1,69 @@
+import DocCard from '@site/src/components/DocCard';
+
+# Roles and Permissions
+
+Windmill provides a roles and permissions system that allows you to control access and manage permissions within your instance and workspaces. Different roles have different levels of access to scripts, flows, apps, and other entities in Windmill.
+
+## Roles in Windmill
+
+1. **Superadmin**: The superadmin role has the highest level of access and can manage the entire Windmill instance, including all its workspaces and invites. Superadmins have full control over the configuration and settings of the Windmill instance.
```

Comment:
The superadmin role has the highest level of access and can manage the entire Windmill instance. Only superadmins can access the dedicated 'admins' workspace. Also by default, superadmins have access to all workspaces as admins of those workspaces. When an admin do an action as a user of a workspace he is not a member of and as such as no username in, his email is used in lieu of the username.